src/services/config-store.js

```diff
@@ -20,10 +20,19 @@ class ConfigStore {
     return this.path.resolve('.', 'forest');
   }
 
+  get schemaDir() {
+    if (!this.lianaOptions?.schemaDir) return null;
+    return this.path.resolve('.', this.lianaOptions.schemaDir);
+  }
```

Comment:
to homogenize and get the default value if the option is not provided
```suggestion
  get schemaDir() {
    if (!this.lianaOptions) return null;

    const { schemaDir } = this.lianaOptions;
    if (schemaDir) {
      return this.path.resolve('.', `${schemaDir}`);
    }

    return this.path.resolve('.', this.projectDirectoryFinder.getAbsolutePath());
  }
```

Reply:
`ensureAbsolutePath` is not executed on the custom schemaDir with this suggestion.

Reply:
let's go without it then.